charts/lakefs/templates/dev-postgresql.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.useDevPostgres }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-secret
+  labels:
+    app: postgres
+data:
+  POSTGRES_DB: postgres
+  POSTGRES_USER: lakefs
+  POSTGRES_PASSWORD: lakefs
+  MAX_CONNECTIONS: "1000"
+---
+apiVersion: apps/v1
+kind: Deployment # Create a deployment
+metadata:
+  name: postgres # Set the name of the deployment
+spec:
+  replicas: 1 # Set 3 deployment replicas
+  selector:
+    matchLabels:
+      app: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      containers:
+        - name: postgres
+          args: ["-c", "max_connections=1000"]
```

Comment:
Looks like we pass max connections from the config and args - I would just keep the default.

Reply:
made it `10`, the default is not good when fluffy and lakefs try to connect at the same time.